lib/srv/regular/sshserver.go

```diff
@@ -805,6 +811,13 @@ func New(addr utils.NetAddr,
 		SessionRegistry: s.reg,
 	}
 
+	if len(s.awsMatchers) != 0 {
+		s.cloudWatcher, err = server.NewCloudServerWatcher(s.ctx, s.awsMatchers, cloud.NewClients())
```

Comment:
For testing purposes shouldn't we receive the cloud clients as`ServerOption`  and use the `cloud.NewClients()` function to configure the cloudClients if not specified in option args received?